Tests/SwiftParserTest/Assertions.swift

```diff
@@ -549,7 +549,11 @@ extension ParserTestCase {
       assertStringsEqualWithDiff(
         "\(mutatedTree)",
         mutatedSource,
-        "A mutation of the original test case failed to round-trip",
+        """
+        A mutation of the original test case failed to round-trip. 
+
+        To debug the failure, add a new `assertParse` test with the mutated source as input.
```

Comment:
You could add this part to `additionalInfo` maybe? Or do you think it could be missed if it was after the diff?

Reply:
Good idea 👍🏽